src/injected/tab-stops-listener.ts

```diff
@@ -131,7 +131,7 @@ export class TabStopsListener {
     @autobind
     private onStopListenToTabStops(): void {
         this.removeListeners();
-        const iframes: NodeListOf<HTMLIFrameElement> = this.getAllFrames();
```

Comment:
I noticed that in other places, `NodeListof<HTMLIFrameElements>` is replaced with `HTMLCollectionOf<HTMLIFrameElements>`, so I am wondering why that is not the case here?

Reply:
This seems outdated, I don't see the explicit type for the const